topgg.rc

```diff
@@ -16,12 +16,12 @@ BEGIN
     BLOCK "040904B0"
     BEGIN
       VALUE "CompanyName", "Top.gg"
-      VALUE "FileDescription", "The official C++ wrapper for the Top.gg API."
-      VALUE "FileVersion", "2.0.0"
-      VALUE "ProductVersion", "2.0.0"
+      VALUE "FileDescription", "A simple API wrapper for Top.gg written in C++."
+      VALUE "FileVersion", "3.0.0"
+      VALUE "ProductVersion", "3.0.0"
```

Comment:
Why is this a major version? did the library break for devs?

Reply:
Some changes here are breaking, like:
- Fetching bot stats now only returns the server count as shards are deprecated.
- The ability to fetch Top.gg users (`get_user`) is removed.